feat: add history.md file from the docs (#269)
hedgedoc/hedgedoc.github.io: content/english/history/details/index.md

---
title: History
---

> If you want to look at a more visual explanation, you can find this [here](/history).

Reviewer: ```suggestion
If you don't like text, there is also a [visual explanation](/history).
```
